### ISSUE TITLE
Exception in "view log", object.replace is not a function

### ISSUE THREAD

**Issue by xiongsongsong** [niusmallnan/kube-explorer#15]
<img width="1282" alt="Screen Shot 2022-07-06 at 23 37 13" src="https://user-images.githubusercontent.com/342509/177589423-f2943649-044e-4557-9c15-d341767ae382.png">

<img width="843" alt="image" src="https://user-images.githubusercontent.com/342509/177589508-bac01224-1271-438b-8a3a-f356ab85a5e7.png">

```shell
# exec
/usr/local/bin/kube-explorer --kubeconfig=/etc/rancher/k3s/k3s.yaml --http-listen-port=9898 --https-listen-port=0 --debug
```

```log
# version
root@fe:~# /usr/local/bin/kube-explorer --version
kube-explorer version v0.2.9 (a81fc99)
root@fe:~# k3s --version
k3s version v1.23.8+k3s1 (53f2d4e7)
go version go1.17.5

```

**Comment by niusmallnan** [comment]
I use the same k3s & kube-explorer combination and I can view logs without those errors.
Could you please provide more detailed steps?

BTW, my browser: 103.0.5060.53 (x86_64)

**Comment by xiongsongsong** [comment]
> 
my browser: Version 103.0.5060.114 (Official Build) (x86_64)

try select "Everything",and reload page. maybe you get that error.

![image](https://user-images.githubusercontent.com/342509/177702570-e85caea8-ddc3-4943-bda6-1a7d82e012df.png)

**Comment by xiongsongsong** [comment]
https://user-images.githubusercontent.com/342509/177707891-1242f56c-22f9-498f-a745-717755ce0a25.mp4

**Comment by ly5156** [comment]
@xiongsongsong 
This problem is caused by this [issue](https://github.com/rancher/dashboard/issues/5979), you can refer to the comment in this [issue](https://github.com/rancher/dashboard/issues/5979#issuecomment-1132530383), temporary solution to this problem, the corresponding pr has been merged, in the next version will be fixed

**Comment by niusmallnan** [comment]
This could be addressed by v0.2.10-rc2.
